openmc/nuclide.py

```diff
@@ -98,7 +99,18 @@ def scattering(self):
     @name.setter
     def name(self, name):
         check_type('name', name, basestring)
-        self._name = name
+
+        if '-' in name:
+            new_name = name.replace('-', '')
+            new_name = new_name.replace('Nat', '0')
+            if new_name.endswith('m'):
+                new_name = new_name[:-1] + '_m1'
+
+            msg = 'OpenMC nuclides follow the GND naming convention. Nuclide ' \
+                  '"{}" is being renamed as "{}".'.format(name, new_name)
+            warnings.warn(msg, DeprecationWarning)
```

Comment:
I think it might be better to have `warnings.warn(msg)` which is by default `UserWarning`. The problem is that DeprecationWarnings are not actually shown unless you turn on a warnings filter for them.